PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingPlacement.kt

```diff
@@ -0,0 +1,42 @@
+package com.braintreepayments.api
+
+/**
+ * Message location within an application
+ * Note: **This module is in beta. It's public API may change or be removed in future releases.**
+ */
+enum class PayPalMessagingPlacement {
+
+    /**
+     * Home view
+     */
+    HOME,
+
+    /**
+     * Category view displaying multiple products
+     */
+    CATEGORY,
+
+    /**
+     * Individual product view
+     */
+    PRODUCT,
+
+    /**
+     * Shopping cart view
+     */
+    CART,
+
+    /**
+     * Checkout view
+     */
+    PAYMENT;
+
+    internal val placementRawValue: String
```

Comment:
```suggestion
    internal val rawValue: String
```
🧶 : take or leave.

Reply:
✅ : https://github.com/braintree/braintree_android/pull/895/commits/ddf36926101cfdf81bcb67b6b5a18dd635bb87b4

Yeah on iOS the messaging SDK has this as an enum but Android just takes in a raw string. Makes sense to update this name since the same enum that exists on iOS doesn't exist in the messaging SDK on Android.